adcm_client/packer/data/config_data.py

```diff
@@ -108,15 +108,19 @@ def _from_remote_git(self, key, **kwargs):
         return value
 
     def _from_tar(self, key, **kwargs):
-        tar = tarfile.open(self.tar) if isinstance(self.tar, str)\
-            else tarfile.open(fileobj=self.tar)
-
         def conf_files(members):
             for tarinfo in members:
                 if fnmatch.fnmatch(tarinfo.name, '*config.y*ml'):
                     yield tarinfo
 
-        confs = conf_files(tar)
+        if isinstance(self.tar, str):
+            conditions = {'name': 'self.tar'}
```

Comment:
In this case, the value should be the variable `self.tar` not a  string `'self.tar'`

Reply:
Add this changes in last commit